frontend/src/embedding-sdk-bundle/components/public/MetabotQuestion/MetabotQuestion.tsx

```diff
@@ -10,10 +10,12 @@ type MetabotQuestionComponent = ((props: MetabotQuestionProps) => ReactNode) & {
 
 export const METABOT_SDK_EE_PLUGIN: {
   MetabotQuestion: MetabotQuestionComponent;
+  MetabotProvider: ({ children }: { children: ReactNode }) => ReactNode;
```

Comment:
Could you extract the type outside? It doesn't seem much atm, but at some point, we had a file with like a crazy amount of props and they were all inline, so it was pretty hard to read.

e.g. what I'm looking for is like
```ts
interface MetabotPlugin {
  MetabotQuestion: MetabotQuestionComponent;
  MetabotProvider: ({ children }: { children: ReactNode }) => ReactNode;
}
```